### PR TITLE
Update hashes for `bash` versions

### PR DESCRIPTION
**Pull request by dav3r**
#  #

## 🗣 Description ##

This PR updates the known hashes for `bash` versions on our current production systems.

## 💭 Motivation and context ##

As our deployed instances move to newer versions of `bash`, we want to update their hashes in this repository so that we can be assured that our code is indeed functioning as intended.

## 🧪 Testing ##

I verified that these updated hashes are indeed being detected when I scanned our Production instances.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

## ✅ Post-merge checklist ##

- [x] Add a tag.

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]
_Bash hash!_

File: src/ioc_scan/ioc_scanner.py [hunk 1]
```diff
@@ -42,8 +42,10 @@
 few things that should hit:
 GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin18)
```
Comment (mcdonnnj) [inline src/ioc_scan/ioc_scanner.py:43]: Just as a thought should we include zsh for macOS since it is the new default?

**Review by mcdonnnj — APPROVED** [review]
Makes sense. I had one thought that you can feel free to decline.

File: src/ioc_scan/ioc_scanner.py (hunk 1 above)
Comment (dav3r) [reply src/ioc_scan/ioc_scanner.py:43]: Yes, that's an excellent idea - added in https://github.com/cisagov/ioc-scanner/pull/42/commits/8925967c197821cc3fd2278b578dcbdc3b180543.

**Pull request merged**